### PR TITLE
Add capistrano tasks to take a node off the load balancer.

### LINKED ISSUE #6538: Add Capistrano task to remove application from the load balancer
## Acceptance Criteria

- [ ] Running `cap --hosts=<hosthere> staging application:remove_from_nginx` will prevent a node from serving traffic on the load balancer
- [ ] Running `cap --hosts=<hosthere> staging application:serve_from_nginx` will allow a node to serve traffic

## Technical Implementation

Add a health monitor that looks at a file on disk and reports a failure if it exists, so that nginx will fail its health check and pull it out.

Example PR: https://github.com/pulibrary/pdc_describe/pull/1985/files

We're really hoping `https://github.com/lbeder/health-monitor-rails/pull/127` will get merged.

## First Step

Update the health-monitor-rails gem so it has the new check.

### PR DESCRIPTION
**Pull request by tpendragon**
Closes #6538

### CODE REVIEW

File: config/deploy.rb [hunk 1]
```diff
@@ -80,6 +80,37 @@ def set_branch
...
+    on roles(:app) do
+      within release_path do
+        execute :touch, "public/remove-from-nginx"
+      end
+    end
+  end
+
+  # You can/ should apply this command to a single host
+  # cap --hosts=figgy-web-staging1.princeton.edu staging application:serve_from_nginx
+  desc "Marks the server(s) to be removed from the loadbalancer"
```
Comment (hackartisan) [inline config/deploy.rb:104]:
```suggestion
  desc "Marks the server(s) to be added back to the loadbalancer"
```

File: config/deploy.rb [hunk 2]
```diff
@@ -80,6 +80,37 @@ def set_branch
...
+  # You can/ should apply this command to a single host
+  # cap --hosts=figgy-web-staging1.princeton.edu staging application:remove_from_nginx
+  desc "Marks the server(s) to be removed from the loadbalancer"
+  task :remove_from_nginx do
+    count = 0
+    on roles(:app) do
+      count += 1
+    end
+    if count > 1
+      raise "You must run this command on individual servers utilizing the --hosts= switch"
```
Comment (hackartisan) [inline config/deploy.rb:93]: Could we allow taking down half the machines at once? especially with figgy, there are a lot of boxes.

**Review by hackartisan — CHANGES_REQUESTED** [review]

File: config/deploy.rb (hunk 2 above)
Comment (tpendragon) [reply config/deploy.rb]: Good idea, implemented.

**Review by hackartisan — APPROVED** [review]
thanks!

**Pull request merged**